<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>377</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configurações</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelLanguage">
     <property name="locale">
      <locale language="Portuguese" country="Brazil"/>
     </property>
     <property name="text">
      <string>Idioma</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbLanguage">
     <property name="locale">
      <locale language="Portuguese" country="Brazil"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkExternalLinks">
     <property name="locale">
      <locale language="Portuguese" country="Brazil"/>
     </property>
     <property name="text">
      <string>Abrir links externos no seu navegador.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelWikiAddress">
     <property name="locale">
      <locale language="Portuguese" country="Brazil"/>
     </property>
     <property name="text">
      <string>Endereço da wiki</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leWikiAddress">
     <property name="locale">
      <locale language="Portuguese" country="Brazil"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelBookmarks">
     <property name="locale">
      <locale language="Portuguese" country="Brazil"/>
     </property>
     <property name="text">
      <string>Favoritos</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="lwFavorites">
       <property name="locale">
        <locale language="Portuguese" country="Brazil"/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnFavoritesDelete">
         <property name="locale">
          <locale language="Portuguese" country="Brazil"/>
         </property>
         <property name="text">
          <string>Deletar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFavoritesAdd">
         <property name="locale">
          <locale language="Portuguese" country="Brazil"/>
         </property>
         <property name="text">
          <string>Adicionar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFavoritesEdit">
         <property name="locale">
          <locale language="Portuguese" country="Brazil"/>
         </property>
         <property name="text">
          <string>Editar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelCustomJS">
     <property name="text">
      <string>JavaScript customizado</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pteCustomJS"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
